backend/substrapp/tasks/tasks_save_image.py

```diff
@@ -51,7 +53,24 @@ def get_task_info(self, args: tuple, kwargs: dict) -> tuple[str, str]:
     # Celery does not provide unpacked arguments, we are doing it in `get_task_info`
     def on_success(self, retval: dict[str, Any], task_id: str, args: tuple, kwargs: dict[str, Any]) -> None:
         function_key, channel_name = self.get_task_info(args, kwargs)
+
+        function_image = FunctionImage.objects.get(function=function_key)
+        # needed because the name is not an optional parameter
+        api_function = ApiFunction.objects.get(key=function_key)
+        orc_function = {
+            "key": str(function_key),
+            # TODO find a way to propagate the name or make it optional at update
+            "name": api_function.name,
```

Comment:
Is this an acceptable workaround? 🤔

Reply:
Why can't we make it optional? I would prefer not creating new links between workers and DB when it is not absolutely necessary

Reply:
I agree it's not a long term solution, just wondering if we want to fix it now, or merge this before it gets bigger and fix as a follow-up 🤔

Reply:
up, did you create the card?

Reply:
Thanks for the reminder 👼 https://linear.app/owkin/issue/FL-1411/name-optional-on-updatefunction-in-orchestrator